x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -31,29 +35,14 @@ export async function getServiceAnomalies({
   const { ml, start, end } = setup;
 
   if (!ml) {
-    logger.warn('Anomaly detection plugin is not available.');
-    return DEFAULT_ANOMALIES;
+    throw Boom.serverUnavailable(ML_ERRORS.ML_NOT_AVAILABLE);
```

Comment:
That's a good idea but I'm going to leave these as-is to avoid too many changes this late. If you really think they're needed mark this PR with changes requested and I can get them in.